pkg/destroy/azure/azure.go

```diff
@@ -205,7 +205,10 @@ func deletePublicRecords(ctx context.Context, dnsClient dns.ZonesClient, records
 
 	zonesPage, err := dnsClient.ListByResourceGroup(ctx, rgName, to.Int32Ptr(100))
 	if err != nil {
-		errs = append(errs, errors.Wrap(err, "failed to list dns zone"))
+		// ignore error if resource group already deleted
+		if !zonesPage.Response().IsHTTPStatus(http.StatusNotFound) {
+			errs = append(errs, errors.Wrap(err, "failed to list dns zone"))
+		}
```

Comment:
for consistency i think we should continue to reuse existing way like https://github.com/openshift/installer/blob/fa6a6b12dbf32441e978ac895cdc4cbd3ea99562/pkg/destroy/azure/azure.go#L379-L381

Reply:
It makes sense to log and return here - so addressed.